packages/indexer-agent/src/__tests__/indexer.ts

```diff
@@ -174,7 +177,7 @@ describe('Indexer tests', () => {
   afterEach(teardown)
 
   // test('Parse Dispute from GraphQL', async () => {})
-  test('Store POI Disputes rejects invalid indexer address', async () => {
+  test('Store PoI Disputes rejects invalid indexer address', async () => {
```

Comment:
Particularly given that the code still uses `POI` instead of `PoI`, and the fact that `PoI` is more arduous to type than `POI`, I'd prefer sticking with `POI`.

Reply:
I've been waffling a bit back and forth on this; happy to stick to `POI`.